doc/en/user/source/configuration/status.rst

```diff
@@ -100,3 +100,203 @@ Module Status Field Descriptions
+     - Number of logical CPUs / cores available
+   * - Number of running process
+     - 316
+     - Total number of process running in the system
+   * - Number of running threads
+     - 1094
+     - Total number of threads running in the system
+   * - CPU load average
+     - 4.12 %
+     - Average load of the CPU in the last second
+   * - CPU * load
+     - 11.43 %
+     - Load of a specific core in the last second
+   * - Used physical memory
+     - 31.58 %
+     - Percentage of the system memory used
+   * - Total physical memory
+     - 31.4 GiB
+     - System total memory
+   * - Free physical memory
+     - 21.4 GiB
+     - System memory available for use
+   * - Used swap memory
+     - 0.00%
+     - Percentage of swap memory used
+   * - Total swap memory
+     - 32.0 GiB
+     - System total swap memory
+   * - Free swap memory
+     - 32.0 GiB
+     - Free swap memory
+   * - File system usage
+     - 65.47 %
+     - File system usage taking in account all partitions
+   * - Partition * used space
+     - 54.8 %
+     - Percentage of space used in a specific partition
+   * - Partition * total space
+     - 338.9 GiB
+     - Total space of a specific partition
+   * - Partition * free space
+     - 117.0 GiB
+     - Free space on a specific partition
+   * - Network interfaces send
+     - 42.0 MiB
+     - Data send through all the available network interfaces
+   * - Network interfaces received
+     - 700.4 MiB
+     - Data received through all the available network interfaces
+   * - Network interface * send
+     - 25.0 MiB
+     - Data send through a specific network interface
+   * - Network interface * received
+     - 250.4 MiB
+     - Data received through a specific network interface
+   * - CPU temperature
+     - 52.00 ºC
+     - CPU temperature
+   * - CPU voltage
+     - 1.5 V
+     - CPU voltage
+   * - GeoServer CPU usage
+     - 3.5 %
+     - Percentage of CPU used by GeoServer in the last second
+   * - GeoServer threads
+     - 49
+     - Number of threads created by GeoServer
+   * - GeoServer JVM memory usage
+     - 5.83 %
+     - Percentage of the JVM memory used by GeoServer
+
+If some information is not available the special term ``NOT AVAILABLE`` will appear. Values will be automatically converted to best human readable unit. 
+
+
+Usage
+^^^^^
+
+The system information will be available in the GeoServer status page in the ``Monitoring`` tab (the following image only shows part of the available system information):
+
+.. figure:: img/gui.png
+   :align: center
+
+|
+
+If the ``Monitoring`` tab is not present, it means that the plugin was not installed correctly. The ``Monitoring`` tab content will be refreshed automatically every second.  
+
+REST interface
+^^^^^^^^^^^^^^
+
+It is possible to request the available system information (monitoring data) through GeoServer REST API. The supported formats are XML, JSON and HTML. 
+
+The available REST endpoints are: ::
+
+    /geoserver/rest/about/monitoring
```

Comment:
Same here, is not monitoring anymore but ``system-status``.

Reply:
Renamed.